components/tcf/secondLayer/src/index.js

```diff
@@ -209,13 +205,12 @@ export default function TcfSecondLayer({
         isOpen={modalOpen}
         closeOnOutsideClick
         closeOnEscKeyDown
-        header={isMobile ? <Logo /> : false}
+        header={<img className={`${CLASS}-logo`} src={logo} alt="logo" />}
```

Comment:
Even you have to render it always, I would let the React.memo, to ensure that the logo that you pass to the modal is always the same. So you can let the logo as a constant with react memo and pass the <Logo /> component in the prop header

Reply:
@maryammdot The use of React.memo is not necessary now since the image is always fixed